RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -34,13 +34,19 @@ enum PackageGroupValidationError: Error {
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 struct TemplateComponentsView: View {
 
-    let paywallComponentsData: PaywallComponentsData
-    let componentViewModel: PaywallComponentViewModel
-    private let onDismiss: () -> Void
+    @Environment(\.userInterfaceIdiom)
+    private var userInterfaceIdiom
```

Comment:
Just for my understanding, why does it need this? Wouldn't the `screenCondition` be enough?